tests/cpgrid/distribution_test.cpp

```diff
@@ -518,6 +518,171 @@ BOOST_AUTO_TEST_CASE(compareWithSequential)
     }
 }
 #endif
+BOOST_AUTO_TEST_CASE(PartitionTest)
+{
+#if HAVE_MPI
+    int size, rank;
+    MPI_Comm_size(MPI_COMM_WORLD, &size);
+    MPI_Comm_rank(MPI_COMM_WORLD, &rank);
+    MPI_Comm twocom = MPI_COMM_WORLD;
+    if (size==1)
+        return;
+    if (size > 2)
+    {
+        MPI_Comm_split(MPI_COMM_WORLD, rank<2, rank, &twocom);
+    }
+    if (rank < 2)
+    {
+        Dune::CpGrid grid(twocom);
+        std::array<int, 3> dims = {{7, 5, 1}};
+        std::array<double, 3> sizes = {{ 1.0, 1.0, 1.0}};
+        std::vector<int> parts = { 0, 1, 1, 0, 0, 0, 0,
+                                   0, 1, 1, 0, 0, 0, 0,
+                                   0, 0, 0, 0, 0, 0, 0,
+                                   0, 0, 0, 0, 0, 0, 0,
+                                   0, 0, 0, 0, 0, 0, 0 };
+        grid.createCartesian(dims, sizes);
+        grid.loadBalance(parts, false, false, 1);
```

Comment:
Can we add here something describing the 3 arguments? For example, /* overlapLayerSize = */ =1. Same for the other test case